### ISSUE TITLE
No OpKernel was registered to support Op 'RandomShuffleQueueV2' with these attrs. 

### ISSUE THREAD

**Issue by tangtaozhanshen** [tensorflow/tensorflow#16266]
I build a graph.pb by python.

```
  classifier = tf.estimator.DNNClassifier(feature_columns=feature_columns,
                                          hidden_units=[60, 60, 60],
                                          n_classes=numbTrainClass,
                                          model_dir=os.path.curdir + "/tmp/app_predict_mode1-12")
```

When I load it on android, 

```
tf = new TensorFlowInterface(mContext.getAssets(), mPbFileName);
...
tf.run(null).
```
This is error.
```
java.lang.IllegalArgumentException: No OpKernel was registered to support Op 'RandomShuffleQueueV2' with these attrs.  Registered devices: [CPU], Registered kernels:
  <no registered kernels>

	 [[Node: enqueue_input/random_shuffle_queue = RandomShuffleQueueV2[capacity=1000, component_types=[DT_INT64, DT_FLOAT, DT_INT32], container="", min_after_dequeue=250, seed=0, seed2=0, shapes=[[], [9], []], shared_name="", _device="/device:CPU:0"]()]]
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tangtaozhanshen** [comment]
Thank! This problem is fixed!  But I still can't retrain weight of neural network in android.

**Comment by poxvoculi** [comment]
Closing because this is an inactive issue of unclear nature.  If you believe there is a bug in TensorFlow, please reopen with a clear description of the suspected bug.

**Comment by karannaik** [comment]
Hey guys,  @tangtaozhanshen @poxvoculi , I am facing the same issue mentioned above.
Below is the error:
`java.lang.IllegalArgumentException: No OpKernel was registered to support Op 'RandomShuffleQueueV2' with these attrs.  Registered devices: [CPU], Registered kernels:
                                                                        <no registered kernels>
                                                                      
                                                                      	 [[Node: enqueue_input/random_shuffle_queue = RandomShuffleQueueV2[capacity=1000, component_types=[DT_INT64, DT_FLOAT, DT_INT32], container="", min_after_dequeue=250, seed=0, seed2=0, shapes=[[], [50,50], []], shared_name=""]()]]
                                                                          at org.tensorflow.Session.run(Native Method)
                                                                          at org.tensorflow.Session.access$100(Session.java:48)
                                                                          at org.tensorflow.Session$Runner.runHelper(Session.java:298)
                                                                          at org.tensorflow.Session$Runner.run(Session.java:248)
                                                                          at org.tensorflow.contrib.android.TensorFlowInferenceInterface.run(TensorFlowInferenceInterface.java:228)
                                                                          at org.tensorflow.contrib.android.TensorFlowInferenceInterface.run(TensorFlowInferenceInterface.java:197)
                                                                          at csulb.beachhacks.asl.TensorflowImageClassifier.recognizeImage(TensorflowImageClassifier.java:136)`


Please let me know, how did you solve the issue. Or still, this issue is to be fixed. Thanks

**Comment by xiaomochen520** [comment]
@tangtaozhanshen Hey, I have a mistake similar to yours. It reminds me that I don't have "CTCLoss". How do you solve your problem? About "RandomShuffleQueueV2"
